### DISCUSSION TITLE
connectivities and distances from seurat to scanpy

### DISCUSSION THREAD

**Discussion by hamidsta** [satijalab/seurat#6010] [Q&A]
Hello , actually i'm analysing a mutliomics dataset , and after running FindMultiModalNeighbors, i want to use the Connectivity and the distance matrix generated to run paga algorithm on it . 
The probleme is that i don't really know, how i should do. By using reticulate i was able to create an anndata object from my seurat object ,but i was not able to import the neighbors . 
Also, here is what paga need to be run : 

connectivities sparse matrix of dtype float32.
Weighted adjacency matrix of the neighborhood graph of data points. Weights should be interpreted as connectivities.

distances sparse matrix of dtype float32.
Instead of decaying weights, this stores distances for each pair of neighbors.

Thank you very much

**Comment by Hikaru16000all** [comment]
Has anyone solved this issue yet?
